Remove ambiguity in kinematics.rst (#745)
ros-planning/moveit2_tutorials: doc/concepts/kinematics.rst

Collision checking is a very expensive operation often accounting for close to 90% of the computational expense during motion planning.
The ``Allowed Collision Matrix`` or ``ACM`` encodes a binary value corresponding to the need to check for collision between pairs of bodies (which could be on the robot or in the world).

> If the value corresponding to two bodies is set to ``true`` in the ACM, it specifies that a collision check between the two bodies is not required.

Reviewer: ```suggestion
If the value corresponding to two bodies is set to ``true`` in the ACM, no collision check between the two bodies will be performed.
```
